draft-packages/select/KaizenDraft/Select/styles.elm.scss

```diff
@@ -37,7 +45,7 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: space-between;
-  min-height: 48px;
+  min-height: $control-height;
```

Comment:
I tried to avoid changing any of the existing CSS (because that would require a lot of QA, and there'd be a risk of ) and this is the only real change I made. It's just replacing `48px` with rem units of `2 * $ca-grid`, so that I could use that to calculate the height minus the borders.